### ISSUE TITLE
[FEEDBACK / FEATURE] [VCC-WEB] Add more ways to Sort & Manage Projects (e.g. Folders)

### ISSUE THREAD

**Issue by DrBlackRat** [vrchat-community/creator-companion#68]
**Issue:**  
Currently there is no way to organizes your Projects and you are forced to sort by name.

**Suggestion::**    
There are multiple things that could / should be done to allow for better organization of projects:  

1. Add a `Last Opened` date / time to each Project  
This should probably be added as another row, like Project Type. It would make it easier to quickly see what Project you had open recently (especially in correlation with 2.)  

2. Allow us to sort our Projects differently  
Currently the our Projects are sorted by Name starting with A, clicking on "Project Name" should revert his, so that it would start with Z. Same thing goes for if Last Opened is added, you should just be able to click on the name of the row to sort your projects with it. (e.g. Opened Recently first and Opened a long time ago first). There should probably also be a small indicator which shows how it's currently sorted. (like an arrow)
This should basically just end up working like the Windows Explorer does.

3. Allow us to create Folders which we could put our Projects in  
This would allow people who do commissions for example to put all of them into one folder and all of their private projects into a different one.

I think all three of these would be great additions to the VCC as they would allow for better organization of our Projects.

**Comment by DrBlackRat** [comment]
Small Update, currently it's sorted by which project got added the last and not by name.

**Comment by BluWizard10** [comment]
Please please, for the love of god, make this happen! Sorting Options is an essential component in a Creator's life.

**Comment by Faxmashine** [comment]
Our QA team gave us the following feedback a while back:

> Users want to organize their projects for ease of use. We should add
> Last open date
> User tags
> Sorting functionality
> Tag search/filter
> Type search/filter

**Comment by DrBlackRat** [comment]
> Our QA team gave us the following feedback a while back:
> 
> > Users want to organize their projects for ease of use. We should add
> > Last open date
> > User tags
> > Sorting functionality
> > Tag search/filter
> > Type search/filter

I agree, a tag system would probably be better than folders. They easier to implement and probably also more useful since you could use multiple tags for one project.
